Cargo.toml

```
@@ -19,7 +19,7 @@ readme = "README.md"
 description = "Retrieve the current user and environment."
 keywords = ["user", "username", "whoami", "platform", "detect"]
 categories = ["os", "wasm", "internationalization", "localization", "value-formatting"]
-include = ["README.md", "src/*"]
+include = ["README.md", "LICENSE_APACHE_2_0.txt", "LICENSE_BOOST_1_0.txt", "LICENSE_MIT.txt", "src/*"]
```

This line has gotten too long, can you make change the formatting, and also keep in alphabetical order?

```suggestion
include = [
    "LICENSE_APACHE_2_0.txt",
    "LICENSE_BOOST_1_0.txt",
    "LICENSE_MIT.txt",
    "README.md", 
    "src/*",
]
```

---

I will sort the list, but after renaming the files the line is shorter and there is not the same need to make it multi line.

(There could perhaps still be benefits in applying the full reformatting that happens anyway when packaging.)